pkg/operator/install.go

```diff
@@ -0,0 +1,87 @@
+package operator
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/openshift-pipelines/release-tests/pkg/client"
+	"github.com/openshift-pipelines/release-tests/pkg/config"
+	"github.com/openshift-pipelines/release-tests/pkg/helper"
+	. "github.com/smartystreets/goconvey/convey"
+	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	op "github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/names"
+	"gotest.tools/v3/icmd"
+	knativetest "knative.dev/pkg/test"
+)
+
+var CR *v1alpha1.Config
```

Comment:
how about descriptive names for exported variables ; e.g.
`Config`, `Clients`

Reply:
That makes sense Really, I am updating it to ```ClusterConfig``` , ```ClientSet```